### PR TITLE
Project Issue Report

### PR DESCRIPTION
**Pull request by vrenaville**
Hello,

This module add a report, to print issue related to a project.
it can be print from the block hours (to be sent with the block hour report in a template by example) or from the project directly.

### CODE REVIEW

File: project_issue_block_report/__openerp__.py [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+
+{
+    'name': 'Project Issue relate block hours',
+    'summary': 'Print Ticket linked to project concerned by block hours',
+    'version': '1.1',
+    'category': 'Project Management',
+    'description': """\
```
Comment (pedrobaeza) [inline project_issue_block_report/__openerp__.py]: Remove `\`

File: project_issue_block_report/i18n/project_issue_block_report.po [hunk 2]
```diff
@@ -0,0 +1,58 @@
+# Translation of OpenERP Server.
```
Comment (pedrobaeza) [inline project_issue_block_report/i18n/project_issue_block_report.po]: Rename to .pot.

File: project_issue_block_report/__openerp__.py [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+Report on Project Issue linked to a project set in invoice line of the related block hours
+""",
+    'author': 'Vincent Renaville',
+    'depends': [
+        'project_issue',
+        'analytic_hours_block'
```
Comment (pedrobaeza) [inline project_issue_block_report/__openerp__.py]: You are not depending on `report_webkit`.

Comment (pedrobaeza) [inline project_issue_block_report/__openerp__.py]: Why it depends on analytic_hours_block if there is any inheritane nor dependency?

**Comment by pedrobaeza** [comment]
Name should include `_webkit` suffix for indicating that includes webkit reports

**Comment by vrenaville** [comment]
Hello,
Thanks for the review !

I have correct (I think) most of errors, 
but I don't understand why to not depend on analytic_hours_block, this report add a report on the model, so I need to inherit from it or I miss something.

Vincent

**Comment by pedrobaeza** [comment]
OK, now I see the dependency. Sorry for not watching it.

:+1:

File: project_issue_block_report_webkit/report/block_project_issue.py [hunk 4]
```diff
@@ -0,0 +1,106 @@
...
+                                                          result_issue_ids,
+                                                          context=self.context)
+            result_records = {
+                'type': type,
+                'issues': result_ids_records,
+                'color': color,
+                }
+            result.append(result_records)
+
+        print str(result)
```
Comment (rdeheele) [inline project_issue_block_report_webkit/report/block_project_issue.py]: i think you can delete this print

**Comment by gurneyalex** [comment]
Travis is red, but the 7.0 branch build is read. 
- no flake8 warning for this module
- no pylint warning for this module
- the server tests in UNIT_TEST mode are green for this module

I'm globally :+1: for this PR (please fix the small problem noted by @rdeheele)

**Comment by rdeheele** [comment]
:+1: thanks Vincent.

**Comment by vrenaville** [comment]
@gurneyalex @rdeheele @pedrobaeza  Thanks for the review.

**Comment by lepistone** [comment]
I'm merging because the red build is unrelated to this module. Thanks!

**Pull request merged**